apsuite/commisslib/meas_ac_orm.py

```diff
@@ -119,13 +146,23 @@ def rf_mode(self, value):
         elif int(value) in self.RFModes:
             self._rf_mode = int(value)
 
+    @property
+    def rf_llrf2use(self):
+        """."""
+        return self._rf_llrf2use
+
+    @rf_llrf2use.setter
+    def rf_llrf2use(self, value):
+        if isinstance(value, str) and value.upper() in {'A', 'B'}:
+            self._rf_llrf2use = value.upper()
+
```

Comment:
I think we should raise an exception here! Otherwise the user may hava a false sense of everything is set correctly.